modules/virt-snocaps-differences.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// * virt/about-virt.doc
+
+:_content-type: CONCEPT
+[id="virt-snocaps-differences_{context}"]
+= Single Node {product-title} differences
+
+You can install {VirtProductName} on a single-node cluster. See xref:../installing/installing_sno/install-sno-preparing-to-install-sno.adoc#install-sno-about-installing-on-a-single-node_install-sno-preparing[About {sno}] to prepare your cluster for installation.
+
+When provisioning an {product-title} {sno} cluster with the assisted installer, out-of-the-box persistent storage is auto-deployed using the {product-title} link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/deploying_openshift_data_foundation_on_single_node_openshift_clusters/index#doc-wrapper[Data Foundation Logical Volume Manager Operator].
```

Comment:
the assisted installer link might make more sense here because everything that follows resides in it's realm

Reply:
Yes, the link should be in the first occurrence of the 'assisted installer' string, so this makes sense.

Reply:
Done.